Needed to fix compilation of the struct module in Cygwin on Python 3.  This
as been fixed upstream for Python 3.7; see https://bugs.python.org/issue21124
diff -r b244bf74b638 Modules/_struct.c
--- a/Modules/_struct.c	Sun Oct 02 13:49:05 2016 +0300
+++ b/Modules/_struct.c	Sun Oct 02 19:54:56 2016 +0900
@@ -2301,6 +2301,9 @@
     if (PyType_Ready(&PyStructType) < 0)
         return NULL;
 
+    if (PyType_Ready(&unpackiter_type) < 0)
+        return NULL;
+
     /* Check endian and swap in faster functions */
     {
         const formatdef *native = native_table;
